config/api.yml

```
@@ -2075,6 +2075,12 @@
         :identifier: instance_reset
       - :name: set_ownership
         :identifier: vm_ownership
+      - :name: retire
+        :identifier:
+        - instance_retire_now
```

Since there's only a single identifier this can just be inlined (same for all of the other sections in this diff as well)

```suggestion
        :identifier: instance_retire_now
```